sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2129,6 +2129,36 @@ pi_result piPluginGetLastError(char **message) {
   return pi2ur::piPluginGetLastError(message);
 }
 
+// Optimization strings
+char EmptyStr[2] = "";
```

Comment:
```suggestion
static const char *EmptyStr = "";
```

Reply:
Made the change for all optimization strings....

Reply:
Resolved. Thanks